.github/actions/publish-nuget/New-NuGetPackage.ps1

```diff
@@ -47,6 +47,7 @@ function Get-ProjectProperty
         Set-Content $temporaryProjectFilePath $updatedProjectFileContent -ErrorAction Stop
 
         $buildOutput = dotnet msbuild $temporaryProjectFilePath /nologo /v:minimal /p:DesignTimeBuild=true /p:BuildProjectReferences=false /t:GetPropertyValue
+        Write-Output "BUILD OUTPUT: '$buildOutput'"
```

Comment:
For whatever reason when I don't have this log here, the main package gets skipped. For example [here](https://github.com/Lombiq/Orchard-Privacy/actions/runs/5753728233). It looks like the new MSBuild based thing sometimes doesn't return anything but I don't know the reason. I assume it's just a coincidence, but it never failed to publish the packages correctly when these logs were present. At any rate if it ever happens again (I expect this to crop up when I continue doing OSOE-603 after this is resolved) it will be invaluable information, so I'd prefer to keep it for now. Is it okay if I open a new bug issue regarding this publication flakyness? It's not related to the current issue so I don't want to hold up the resolution here.

Reply:
This is some dark magic shit. Yeah, then please keep it here, with a comment on why, and indeed create an issue and link it in the comment. We'll keep that low prio though, since having the build output in the workflow output won't bother us.

Reply:
Added https://github.com/Lombiq/GitHub-Actions/issues/250